packages/bpk-component-graphic-promotion/src/BpkGraphicPromo.module.scss

```
@@ -33,227 +45,215 @@
   color: $bpk-color-white;
   box-shadow: none;
 
-  @include breakpoint-landscape {
-    &__sponsor-label {
-      margin-bottom: bpk-spacing-sm();
+  @include bpk-breakpoint-small-tablet {
+    @include aspect-ratio(133%);
+  }
 
-      @include bpk-label-1;
-    }
+  /* remove shadow and hightlights from Backpack Card */
+  &::after {
+    display: none;
+  }
 
-    &__sponsor-content {
-      margin-bottom: bpk-spacing-xl();
-    }
+  &__container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
 
-    &__tagline {
-      margin-bottom: bpk-spacing-md();
+    @include bpk-breakpoint-above-tablet {
+      height: 28.75rem;
 
-      @include bpk-label-1;
-    }
+      @include bpk-margin-leading(bpk-spacing-xxxl(), false);
+      @include bpk-margin-trailing(bpk-spacing-xxl(), false);
 
-    &__subheading {
-      @include bpk-body-default;
+      /* adjusting for 1.5rem difference between margins */
+      &--center > * {
+        @include bpk-margin-trailing(bpk-spacing-lg());
+      }
     }
 
-    &__headline {
-      @include breakpoint-landscape-desktop {
-        @include bpk-hero-5;
+    @include bpk-breakpoint-tablet {
+      height: 22.5rem;
+
+      @include bpk-margin-leading(bpk-spacing-xxl(), false);
+      @include bpk-margin-trailing(bpk-spacing-xl(), false);
+
+      /* adjusting for 1.5rem difference between margins */
+      &--center > * {
+        @include bpk-margin-trailing(bpk-spacing-md());
       }
+    }
+
+    @include bpk-breakpoint-small-tablet {
+      padding: bpk-spacing-xl();
+      justify-content: space-between;
+      align-items: initial;
 
-      @include breakpoint-landscape-tablet {
-        @include bpk-heading-1;
+      @include bpk-margin-leading(0, true);
```

You shouldn't need to pass true as the value as it defaults to true :) 
```suggestion
      @include bpk-margin-leading(0);
```

---

I just wanted to be explicit, but I removed it now. 🙂